server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -124,8 +124,7 @@ export default class ProbationPractitionerReferralsController {
     const existingDraftReferrals = await this.interventionsService.getDraftReferralsForUserToken(accessToken)
 
     const structuredInterventionsFileDownloadPaths = {
-      xlsx: 'assets/downloads/Structured interventions list v1_3_3.xlsx',
-      pdf: 'assets/downloads/Structured interventions list v1_3_3.pdf',
+      xlsx: 'assets/downloads/Structured interventions.xlsx',
```

Comment:
Assuming people download this and we update it after -- can we help people understand that they have an outdated version?

Reply:
Spoken with Sarah on this - I've added versioning back in so it's now v1.3.4 to show the user that they are using a newer copy. In terms of bookmarking the file, not sure how this could work unless we forever more wanted to hard-code 1.3.3 in which doesn't seem like a good approach?